src/stock-on-hand-mobile/pages/program-select.jsx

```diff
@@ -0,0 +1,116 @@
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+import React, { useMemo, useEffect, useState } from 'react';
+import { useSelector } from "react-redux";
+import { RadioButton } from '../components/radio-button';
+
+import Select from '../../react-components/inputs/select';
+
+
+const ProgramSelect = ({ offlineService }) => {
+
+    const facility = useSelector(state => state[`facilitiesStockOnHand`][`userHomeFacilityStockOnHand`]);
+    
+    const programs = facility.supportedPrograms.map(({ id, name }) => ({ value: id, name }));
+
+    const menu = document.getElementsByClassName("header ng-scope")[0];
+    
+    const [facilityId, setFacilityId] = useState(null);
+    const [programId, setProgramId] = useState(null);
+    const [programName, setProgramName] = useState(null);
+    const [facilityType, setFacilityType] = useState("MyFacility");
+
+    const radioChangeHandler = (e) => {
+        setFacilityType(e.target.value);
+    };
+
+    useEffect(() => menu.style.display = "", [menu]);
+
+    return (
+        <div>
+            <div className="page-header-responsive">
+                <h2 id='program-select-header'>Stock on Hand</h2>
+            </div>
+            <div className="page-content">
+                <label 
+                  id='facility-type-header' 
+                  style={{marginBottom: "4px", fontFamily: "Arial", fontSize: "16px"}}
+                >
+                  Facility Type
+                </label>
+                <div>
+                    <RadioButton
+                        changed={radioChangeHandler}
+                        id="1"
+                        isSelected={facilityType === "MyFacility"}
+                        label={`My Facility `}
+                        additionalInfo={`(${facility.name})`}
+                        value="MyFacility"
+                    />
+                    <RadioButton
+                        changed={radioChangeHandler}
+                        id="2"
+                        isSelected={facilityType === "SupervisedFacility"}
+                        label="Supervised Facility"
+                        value="SupervisedFacility"
+                    />
+                </div>
+                <div style={{marginTop: "8px", marginBottom: "8px"}}>
+                    <div className='required' style={{marginBottom: "4px", fontFamily: "Arial", fontSize: "16px"}}>
+                      <label 
+                        id='facility-type-header'
+                      >
+                        Program
+                      </label>
+                    </div>
+                    <div className='field-full-width' style={{marginBottom: "8px"}}>
```

Comment:
Why className='field-full-width' is in single quote? I also saw above and below as well not only for className but also for id. It is better to stick to one notation than to mix between single and double quote. For html I see and use the double notation myself while for the logic, single.